Firestore/Source/Local/FSTLRUGarbageCollector.mm

```diff
@@ -17,16 +17,30 @@
 #import "Firestore/Source/Local/FSTLRUGarbageCollector.h"
 
 #include <queue>
+#include <utility>
 
 #import "Firestore/Source/Local/FSTMutationQueue.h"
+#import "Firestore/Source/Local/FSTPersistence.h"
 #import "Firestore/Source/Local/FSTQueryCache.h"
+#include "Firestore/core/include/firebase/firestore/timestamp.h"
 #include "Firestore/core/src/firebase/firestore/model/document_key.h"
+#include "Firestore/core/src/firebase/firestore/util/log.h"
 
+using firebase::Timestamp;
 using firebase::firestore::model::DocumentKey;
 using firebase::firestore::model::ListenSequenceNumber;
 
+const long kFIRFirestorePersistenceCacheSizeUnlimited = -1;
 const ListenSequenceNumber kFSTListenSequenceNumberInvalid = -1;
 
+static long toMilliseconds(const Timestamp &ts) {
+  return (1000 * ts.seconds()) + (ts.nanoseconds() / 1000000);
+}
+
+static long millisecondsBetween(const Timestamp &start, const Timestamp &end) {
+  return toMilliseconds(end) - toMilliseconds(start);
```

Comment:
You should be able to do arithmetic on Timestamps as std::chrono::time_points. See https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/include/firebase/firestore/timestamp.h#L140

I think it should be something like:

```c++
duration_cast<std::chrono::milliseconds>(end.ToTimePoint() - start.ToTimePoint());
```

Reply:
Done. Now returns `Millis::rep`, which seems good enough for logging.